kratos.gid/kratos.unix.bat

```diff
@@ -20,16 +20,16 @@ fi
 # and maintains OLD_LD_LIBRARY_PATH with previous settings
 # therefore, we use the OLD_LD_LIBRARY_PATH and prepend the path to the kratos libs
 if [ "$OLD_LD_LIBRARY_PATH" != "" ]; then
-    export LD_LIBRARY_PATH="$3/exec/Kratos":"$3/exec/Kratos/libs":$OLD_LD_LIBRARY_PATH
+    export LD_LIBRARY_PATH="$3/exec/Kratos/bin/Release":"$3/exec/Kratos/bin/Release/libs":$OLD_LD_LIBRARY_PATH
```

Comment:
👎

Reply:
ouch! That slipped in!! Will remove it immediately